src/tool/subcommands/api_cmd.rs

```diff
@@ -50,9 +51,20 @@ pub enum ApiCommands {
         #[arg(long, value_enum, default_value_t = RunIgnored::Default)]
         /// Behavior for tests marked as `ignored`.
         run_ignored: RunIgnored,
+        /// Maximum number of concurrent requests
+        #[arg(long, default_value = "8")]
+        max_concurrent_requests: usize,
     },
 }
 
+struct ApiTestFlags {
```

Comment:
Let's document this. I have no clue what, for example, a `String` `filter` is and what does it expect.

Reply:
we already have a short description for these flags under `ApiCommands`
https://github.com/ChainSafe/forest/blob/ced0f7570ccc9e52b01db3ddf56dc0439288da91/src/tool/subcommands/api_cmd.rs#L41-L52

Reply:
I wasn't aware of that. Perhaps adding a Rust doc link to the `ApiCommands` struct would help future readers. What do you think?

Reply:
yes, i'll add